### ISSUE TITLE
Running ":" as command -> IndexError

### ISSUE THREAD

**Issue by The-Compiler** [The-Compiler/qutebrowser#1690]
Since #1577, when doing `:<Enter>` or `qutebrowser --debug --temp-basedir ': '` we get:

```
Traceback (most recent call last):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/app.py", line 202, in _process_args
    process_pos_args(args.command)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/app.py", line 262, in process_pos_args
    commandrunner.run_safely_init(cmd[1:])
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 304, in run_safely_init
    self.run(text, count)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 274, in run
    for result in self.parse_all(text):
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 123, in parse_all
    text = self._get_alias(text, text)
  File "/home/florian/proj/qutebrowser/git/qutebrowser/commands/runners.py", line 97, in _get_alias
    alias = config.get('aliases', parts[0])
IndexError: list index out of range
```

@mgoral do you want to fix that one? :wink:

**Comment by The-Compiler** [comment]
Already three crash reports for this, I guess I'll need to do a small v0.8.1 release for this today.

**Comment by mgoral** [comment]
@The-Compiler: sorry, I haven't seen a notification from github till now. As I understand you fixed it for me, right? :)

**Comment by The-Compiler** [comment]
Yep - I fixed it in a84807ed05115fb690f300ab77f1742f577c907c and did a v0.8.1 release as a lot of crash reports came in :wink:

**Comment by mgoral** [comment]
Thanks! And I wondered what jackass made you release such quick update. ;)
